numba/typed/typedlist.py

```diff
@@ -377,7 +381,7 @@ def reverse(self):
     def copy(self):
         return _copy(self)
 
-    def index(self, item, start=None, stop=None):
+    def index(self, item: T, start=None, stop=None):
```

Comment:
Type annotations for `start` and `stop`?

Reply:
fixed